CMake doesn't find_path ctags properly when crossbuilding.
The CTAGS_SUBMODULE variable is never used though,
and ctags are still built and installed properly
diff --git a/submodules/CMakeLists.txt b/submodules/CMakeLists.txt
index 7808c7628f..5b3862e314 100644
--- a/submodules/CMakeLists.txt
+++ b/submodules/CMakeLists.txt
@@ -3,11 +3,6 @@ cmake_minimum_required(VERSION 3.18)
 # ######################################################################################################################
 # Check submodules are here
 # ######################################################################################################################
-find_path(CTAGS_SUBMODULE CMakeLists.txt PATHS ${CMAKE_CURRENT_SOURCE_DIR}/ctags/)
-if(NOT CTAGS_SUBMODULE)
-    message(STATUS "Could not locate `${CMAKE_CURRENT_SOURCE_DIR}/ctags/CMakeLists.txt` file")
-    message(FATAL_ERROR "Please make sure you have run `git submodule update --init`")
-endif()
 
 # ######################################################################################################################
 # cJSON
